middleware/middleware.go

```diff
@@ -0,0 +1,46 @@
+package middleware
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/labstack/gommon/log"
+	"time"
+)
+
+func Create() *echo.Echo {
+	e := echo.New()
+
+	e.Use(middleware.CSRF())
+
+	e.Use(middleware.TimeoutWithConfig(middleware.TimeoutConfig{
+		ErrorMessage: "Uh Oh! You Timed Out Bud!",
+		OnTimeoutRouteErrorHandler: func(err error, c echo.Context) {
+			log.Print(c.Path())
```

Comment:
This log statement won't be very helpful since it is only the path with no information about the timeout.